app/models/aid_application.rb

```diff
@@ -183,6 +183,8 @@ class AidApplication < ApplicationRecord
     validates :receives_calfresh_or_calworks, inclusion: { in: [true, false] }
     validates :racial_ethnic_identity, presence: true
     validates :attestation, inclusion: { in: [true], message: I18n.t('activerecord.errors.messages.attestation') }
+    validates :valid_work_authorization, inclusion: { in: [false], message: I18n.t('activerecord.errors.messages.work_authorization')}
```

Comment:
please verify, but when doing validation messages for strings in `activerecord.errors.messages` you can replace the I18n with just the final string name. e.g. :
```suggestion
    validates :valid_work_authorization, inclusion: { in: [false], message: :work_authorization }
```

It looks like we didn't do that consistently, but wanted to flag for your learning.

Also I think if you rebase off of master you'll find the strings now currently exist in the translation file.

Reply:
sweet! Verified this is true and changed it other places in aid_application.rb to be consistent